LICENSE

```diff
@@ -187,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2014 The Kubernetes Authors.
+   Copyright 2018 The kube-rs Authors.
```

Comment:
```suggestion
   Copyright 2018-2021 The kube-rs Authors.
```

Reply:
bumping years in copyright files isn't actually a legally viable nor necessary thing to do: https://matija.suklje.name/how-and-why-to-properly-write-copyright-statements-in-your-code#why-keep-the-year

Reply:
you could also go the [client-go way](https://github.com/kubernetes/client-go/blob/master/LICENSE#L190)